scss/_nav.scss

```diff
@@ -182,8 +209,8 @@
 
 .tab-content {
   // Boosted mod
-  padding: $spacer subtract($nav-tabs-link-padding-x, $nav-tabs-border-width);
-  border: $nav-tabs-border-width solid;
+  padding: $spacer subtract(var(--#{$prefix}nav-tabs-link-padding-x), var(--#{$prefix}nav-tabs-border-width));
```

Comment:
Not defined in scope -> Need to see what we can do

Reply:
I'm letting the content as it is in _main_ branch. It is safer than using undefined nav tabs CSS var in this scope. Maybe the solution could be to have specific tab content CSS vars.